examples/icecream/tests/tests.rs

```diff
@@ -38,6 +38,7 @@ fn test_sunny_day() {
                     .expect("Serialize account address."),
             },
         )
+        .print_emitted_events()
```

Comment:
**Suggestion (non-blocking):** Maybe add a comment to what this line does, for educational purpose.

Reply:
Added